tests/test_forward_ref.py

```diff
@@ -44,6 +44,9 @@ def module():
         class Foo(BaseModel):
             a: 'Bar' = None
 
+            class Config:
+                warn_on_undefined_types = False
+
```

Comment:
This is reused many times in the test. Perhaps create a new BaseClass with the proper config and inherit? (Could make tests dry-er)